pkg/server/server.go

```diff
@@ -418,6 +447,7 @@ func (s *Server) Check(ctx context.Context, req *openfgav1.CheckRequest) (*openf
 		attribute.KeyValue{Key: "user", Value: attribute.StringValue(tk.GetUser())},
 	))
 	defer span.End()
+	start := time.Now()
```

Comment:
Put this before the trace. If the trace setup takes a while it should be representative.